### DISCUSSION TITLE
Docker's fine (I guess), but, airflow

### DISCUSSION THREAD

**Discussion by arisdelacruz** [apache/airflow#33151] [Q&A] [answered]
Hello. I hope someone can help me. So, the docker's fine (I guess), but, the airflow deployment:

![2023-07-21_00-11-43-8dc1f1f56806bb417eb66cf1be769c6a](https://github.com/apache/airflow/assets/115809819/7d1a4956-9157-4f36-9157-b159f961316d)

and, when deployed:

![2023-07-18_16-20-34-6e908c8f6d36992569eb37637a69b5a6](https://github.com/apache/airflow/assets/115809819/9067452c-c3d8-4d29-a56e-44d5ed39dd76)

I'm wondering if there's any positive act that I need to do on this matter.

Thanks

**Comment by potiuk** [comment, marked as answer]
Looks like your port is not forwarded from your localhost, maybe you need to specify the real IP address instead of localhost?. I have no idea how this works on Windows, but you might want to look for simmilar issues for port forwarding from docker on windows. Docker on Windows works in the way that it runs on a separate virtual machine, so maybe you need to connect there..

Wild guessing - but it does not seem an Airflow isssue, look for similar Docker/Windows/Port issues

**Reply by arisdelacruz** [reply]
@potiuk 

thank you
